doc/classes/Dictionary.xml

```diff
@@ -4,16 +4,60 @@
 		Dictionary type.
 	</brief_description>
 	<description>
-		Dictionary type. Associative container which contains values referenced by unique keys. Dictionaries are always passed by reference.
+		Dictionary type. Associative container which contains values referenced by unique keys. Just like in real life each dictionary composes multiple pairs of unique keys and values. Each pair is created by joining them via colon [code]Key:Value[/code].
 		Erasing elements while iterating over them [b]is not supported[/b].
 		Creating a dictionary:
 		[codeblock]
-		var d = {4: 5, "A key": "A value", 28: [1, 2, 3]}
+		var my_dir = {} # Creates an empty dictionary
+		var points_dir = {"White" : 50,"Yellow" : 75, "Orange" : 100}
+		[/codeblock]
+		You can access values of a dictionary by refrencing approrpiate key in above example [code]points_dir["White"][/code] would return value of 50.
+		[codeblock]
+		export(String, "White","Yellow", "Orange") var my_color
+		var points_dir = {"White" : 50,"Yellow" : 75, "Orange" : 100}
+		
+		func _ready():
+		    var points = points_dir[my_color]
+		[/codeblock]
+		In above code [code]var points[/code] will be assigned value that is paired with appropriate color selected in [code]var my_color[/code]
+		Dictionaries can contain more complex data 
+		[codeblock]
+		my_dir = {"First Array" : [1,2,3,4]} # Assigns Array to a Key
 		[/codeblock]
 		To add a key to an existing dictionary, access it like an existing key and assign to it:
 		[codeblock]
-		d[4] = "hello"  # Add integer 4 as a key and assign the String "hello" as its value.
-		d["Godot"] = 3.01  # Add String "Godot" as a key and assign the value 3.01 to it.
+		var points_dir = {"White" : 50,"Yellow" : 75, "Orange" : 100}
+		var points_dir["Blue"] = 150  # Add "Blue" as a key and assign 150 as its value.
+		[/codeblock]
+		Finally dictionaries can contain different types of keys and values in the same dictionary
+		[codeblock]
+		var my_dir = {"String Key" : 5, 4 : [1,2,3], 7 : "Hello"} # is a valid dictionary
+		[/codeblock]
+		Below methods allow you to perform more operations on dictionaries
```

Comment:
Other classes do not contain such a sentence.

Reply:
removed